src/renderer/corporate/member/corporate-member-form.jsx

```diff
@@ -75,6 +75,8 @@ class CorporateAddMemberComponent extends PureComponent {
 			classes,
 			onFieldChange = () => {},
 			availablePositions = [],
+			positions = new Set(),
```

Comment:
maybe better to use plane data structure

Reply:
Yeah, set was bad choice here.
Used it initially when creating the UI component and then tried to adapt around it when wiring-up. Switched to a plain array in 6913f4e